doc-review-qs copy edit of quick starts for ModelMesh

MelissaFlinn opened the pull request:
#### Motivation
Copy edit requested by development

#### Modifications
Multiple updates to the readme files

#### Result


#### PR checklist

Checklist items below are applicable for development targeted to both fast and stable branches/tags
- [ ] Unit tests pass locally
- [ ] FVT tests pass locally
- [ ] If the PR adds a new container image or updates the tag of an existing image (not build within cpaas), is the corresponding change made in live-builder and cpaas-midstream to add/update the image tag in the operator CSV? Link the PRs if applicable

Checklist items below are applicable for development targeted to both fast and stable branches/tags
- [ ] Tested modelmesh serving deployment with odh-manifests and ran odh-manifests-e2e tests locally

MelissaFlinn commented:
Needs a review by @Jooho

MelissaFlinn commented:
My apologies - this PR needs a review by Jooho Lee before it is merged and so I changed the status to DRAFT

Jooho commented on opendatahub/docs/get-started-odh-modelserving.md:
```diff
@@ -1,29 +1,30 @@
+## Prerequisites
 
-- **OpenShift cli** - The installation will occur via the terminal using [oc](https://docs.openshift.com/container-platform/4.11/cli_reference/openshift_cli/getting-started-cli.html#cli-installing-cli_cli-developer-commands).
+- You have `cluster-admin` or `dedicated-admin` access to an OpenShift cluster.
```
Just found that `or dedicated-admin` needed to be removed. please remove this part.

Jooho commented on opendatahub/quickstart/README.md:
```diff
@@ -1,28 +1,30 @@
-The folder structure for the Quickstart is as follows:
+- [Sample Model Deployment](./basic/README.md)
+- [Sample Model Deployment and Autoscaler](./hpa/README.md)
+- [Sample Model Deployment and Persistent Volume Claim](./pvc/README.md) 
```
Sample Model Deployment and Persistent Volume Claim would be like this
Sample Model Deployment using Persistent Volume Claim

Jooho commented on opendatahub/quickstart/common_docs/modelmesh-install.md:
```diff
@@ -1,26 +1,45 @@
 ~~~
 source ../env.sh
+
 cd opendatahub/quickstart/basic
```
This folder can be changeable so I think changing it to this would be better. What do you think?
```
cd opendatahub/quickstart/$QUICK_START_FOLDER   # (ex) cd opendatahub/quickstart/basic  
```

Jooho commented on opendatahub/quickstart/pvc/README.md:
```diff
@@ -1,22 +1,12 @@
-# Quick Start Guide - PVC Feature
+# Quick Start - Sample Model Deployment and Persistent Volume Claim
```
sync with the title in another doc.
~~~
# Quick Start - Sample Model Deployment using Persistent Volume Claim
~~~

Jooho commented:
@MelissaFlinn Thanks for your review!! I leave some comments on the files. Please take a look at them.
Plus, could you add your signature to the commit. This is how you can do
~~~
oc commit -s -m "XXX"
~~~

MelissaFlinn commented:
Thank you @Jooho - I addressed your comments and added my signature to the second commit.
This PR is ready for merging after approval by the project owners. Thank you

Jooho reviewed (approved):
/lgtm
